### ISSUE TITLE
Index entries in exercise go missing from LaTeX

### ISSUE THREAD

**Issue by Alex-Jordan** [rbeezer/mathbook#1302]
An `idx` inside an `exercise` currently doesn't do anything for LaTeX output. Index entries simply go missing from print. Some discussion at #1203.

**Comment by kcrisman** [comment]
Similarly for `exercisegroup`, I believe.  

You also had some sample code there; here is what I use as a workaround:
```
diff --git a/xsl/pretext-latex.xsl b/xsl/pretext-latex.xsl
index 96d0b55..2f98d16 100644
--- a/xsl/pretext-latex.xsl
+++ b/xsl/pretext-latex.xsl
@@ -5426,6 +5426,7 @@ along with MathBook XML.  If not, see <http://www.gnu.org/licenses/>.
     <!-- depending on what type of division the "exercise" lives in. -->
     <!-- For each, exactly one "ancestor" is true, and then the      -->
     <!-- expression will evaluate to the corresponding global switch -->
+    <xsl:apply-templates select="idx"/>
     <xsl:variable name="b-has-statement" select="true()"/>
     <xsl:variable name="b-has-hint"
         select="(ancestor::exercises and $b-has-divisional-hint) or
```

**Comment by rbeezer** [comment]
I've got a pile of index-related work to do, tagged now so I'll see it then.

**Comment by kcrisman** [comment]
Hopefully resolved by #1419 also exercise groups.

**Comment by rbeezer** [comment]
Fixed at #1419
